source/about/cloud-shared.rst

```diff
@@ -0,0 +1,85 @@
+Cloud Shared
```

Comment:
similarly may we can add a high level architecture

Reply:
@spirosoik - Yes. Is one available?